scripts/revng-check-conventions

```diff
@@ -70,7 +73,9 @@ USE_CLANG_FORMAT_FILE="0"
 
 PRINT_CLANG_FORMAT_CONFIG="0"
 
-while [[ $# > 0 ]]; do
+GIT_LS_FILES="git ls-files -om --exclude-standard"
```

Comment:
Drop `-o`, we only want to make changes to git-tracked files.

Reply:
Done